cpp/modmesh/view/RCameraController.hpp

```diff
@@ -128,77 +126,95 @@ class RCameraInputListener : public Qt3DCore::QEntity
     Qt3DInput::QButtonAxisInput * m_keyboard_tz_neg_input;
 }; /* end class RCameraInputListener */
 
-class CameraController
+class CameraController : public Qt3DExtras::QAbstractCameraController
```

Comment:
This needs to be named as `RCameraController` because it is a Qt class customized (derived) in modmesh.  See https://github.com/solvcon/modmesh/blob/master/STYLE.rst#qt for the modmesh Qt style.